clients/src/main/java/org/apache/kafka/clients/tools/VerifiableProducer.java

```diff
@@ -124,6 +132,23 @@ private static ArgumentParser argParser() {
                 .choices(0, 1, -1)
                 .metavar("ACKS")
                 .help("Acks required on each produced message. See Kafka docs on request.required.acks for details.");
+        
+        parser.addArgument("--producer-prop")
```

Comment:
The only place I kinda see this is in kafka-topic.sh, but neither KIP-4 nor KIP-14 make reference to allowing a list of `prop=value`.

Currently kafka-topic.sh doesn't really have this behavior (should it?...):
This works (one --config per config):
```
kafka_fork $ bin/kafka-topics.sh --topic abc --zookeeper=localhost:2181 --create --partitions 1 --replication-factor 1 --config max.message.bytes=111 --config flush.ms=222
```

This silently ignores the second config (flush.ms):
```
kafka_fork $ bin/kafka-topics.sh --topic abc --zookeeper=localhost:2181 --create --partitions 1 --replication-factor 1 --config max.message.bytes=111 flush.ms=222
```

From a quick scan of https://reviews.apache.org/r/34554/diff/7#3 it looks like this behavior is unchanged in the KIP-4 patches.

That all said, I changed the name to "--config" to be consistent.